Transaction.proto

```diff
@@ -0,0 +1,121 @@
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Network.Protocol.Protobuffs <https://github.com/catalyst-network/protocol-protobuffs>
+*
+* Catalyst.Network.Protocol.Protobuffs is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+* 
+* Catalyst.Network.Protocol.Protobuffs is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+* 
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Network.Protocol.Protobuffs If not, see <https://www.gnu.org/licenses/>.
+*/
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package Catalyst.Protocol.Transaction;
+
+/**
+ * Transaction
+ *
+ * Version: 1 for non-confidential transaction, 2 for confidential transaction
+ * TimeStamp:
+ * TransactionFees: 8 bytes, clear text, fees * 10^12 - always positive
+ * LockTime: 32 bits
+ * STEntries: 0 field for confidential transaction
+ * CFEntries: 0 field for standard transaction
+ * Signature:
+ * EntryRangeProof: 0 field for standard transaction
+ */
+message Transaction {
+    uint32 Version = 1 [default = 1];
```

Comment:
is this default=2 same as the comment from @monsieurleberre regarding deltas? Shall I remove? can we doc what version field implies. Thanks